Grant Application - Reference blockchain system.md

```diff
@@ -0,0 +1,286 @@
+
+![img](https://github.com/hoho-yin/Grants-Program/blob/hoho-yin-patch-1/deliverables.png)
+
+We have made an inital design on the standard format of the token and smart contract of our proposed system as follows. More specific details are still planning now, and we will continue working to finish the design and development after the grant is given.
+
+**Pallets Token standard:**
+
+| Key | Value | Attribute | Operator |
+| --: | ----- | --------- | -------- |
+| Version | _versionId | compulsory | automatic |
+| Work | _tokenId | compulsory | automatic |
+| Creator  | _address | invariant | creator |
+| Reference | _tokenId | multiple | creator |
+| Content | name, symbol, tokenURI | customized | creator |
+| Transaction | Transaction_id | multiple | automatic |  
+
+Each token will represent a work with unique token id and version id by recording its content and creator especially the reference relationship with other works. The earnings of the work like being purchased and invested will be recorded within the token, and distributed with all its referred tokens via transactions. 
+
+**ink! smart contract standard:**
+
+| Function | Input | Logic | Output |
+| -------: | ----- | ----- | ------ |
+| Mint | address,content,reference | Represent the new work and record the referring works of the new work | token_id |
+| Referred | address, token_id | Set the revenue distribution scheme when being referred by following works |
+| Append | address,token_id,reference,content  | Can’t modify original items |  |
+| Transfer | address,token_id | Change the ownership of one’s token | address, transaction |
+
+When creating a new work, a corresponding token will be minted automatically with unique token id to represent the new work and store all its referring works. The token also defines when it's referred by the following new tokens, how the earnings of the new work should be distributed with it.
+
+## Development Roadmap :nut_and_bolt:
+
+**Expected deliverables:**
+
+1. A reference token standard that records and reflects the nominal reference relationship between new work and its referred old work. When creating a new work, the corresponding new token will also be generated and attached where the reference relationship between the new work and its referred work will be added and stored. 
+
+2. A reference system that makes old work truly composable and interoperable with the new work. By referring to the old work, new work could be created by adding anything new or changing anywhere on the referred work while still connecting and sharing other parts of the referred work seamlessly. 
+
+3. A revenue allocation and distribution system that confirms the revenue distribution scheme agreed by different works with the reference relationship. It will track the earnings of the new work, and distribute them with the old work which the new work refers to according to their distribution scheme.
+
+All programs will be developed in !ink on Pallets.
```

Comment:
Smart contracts don't require any pallets, other than the `contracts` pallet. I suppose you mean "for parachains with `contracts` pallet?